src/foam/box/DedicatedWorkerRegistry.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.box',
+  name: 'DedicatedWorkerRegistry',
+  extends: 'foam.box.ProxyBox',
+  implements: [ 'foam.box.BoxRegistryBox' ],
```

Comment:
I naturally thought that this is more of an extension of `foam.box.BoxRegistryBox` and it implements `foam.box.ProxyBox` but doing so is not possible with importing registry as well.

(Additionally, most places tend to extend `foam.box.ProxyBox`, so I believe this should be the right way of doing this)